py/server/deephaven/stream/table_publisher.py

```diff
@@ -0,0 +1,123 @@
+#
+# Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+#
+"""The table_publisher module supports publishing Deephaven Tables into blink Tables."""
+
+import jpy
+import traceback
+
+from typing import Callable, Dict, Optional
+
+from deephaven._wrapper import JObjectWrapper
+from deephaven.column import Column
+from deephaven.dtypes import DType
+from deephaven.execution_context import get_exec_ctx
+from deephaven.jcompat import j_runnable
+from deephaven.table import Table
+from deephaven.update_graph import UpdateGraph
+
+_JTableDefinition = jpy.get_type("io.deephaven.engine.table.TableDefinition")
+_JTablePublisher = jpy.get_type("io.deephaven.stream.TablePublisher")
+_JRuntimeException = jpy.get_type("java.lang.RuntimeException")
+
+
+class TablePublisher(JObjectWrapper):
+    """Supports publishing Table data."""
+
+    j_object_type = _JTablePublisher
+
+    def __init__(self, j_table_publisher: jpy.JType):
+        self.j_table_publisher = j_table_publisher
+
+    @property
+    def j_object(self) -> jpy.JType:
+        return self.j_table_publisher
+
+    @property
+    def is_alive(self) -> bool:
+        """Checks whether this publisher is alive; if False, the publisher should stop publishing new data and release
+        any related resources as soon as practicable since publishing won't have any downstream effects.
+
+        Once this is False, it will always remain False. For more prompt notifications, publishers may
+        prefer to use on_shutdown_callback during construction.
+
+        Returns:
+            if this publisher is alive
+        """
+        return self.j_table_publisher.isAlive()
+
+    def add(self, table: Table):
```

Comment:
```suggestion
    def add(self, table: Table) -> None:
```
I think we have been indicating no return values.